ee/debug/checkups/services_windows.go

```diff
@@ -228,15 +232,46 @@ func gatherServices(z *zip.Writer, serviceManager *mgr.Mgr) error {
 	return nil
 }
 
+// gatherServiceManagerEvents uses Get-WinEvent to fetch the service manager logs. This might be newer than Get-EventLog
+func gatherServiceManagerEvents(ctx context.Context, z *zip.Writer) error {
+	out, err := z.Create("eventlog-Get-WinEvent.json")
+	if err != nil {
+		return fmt.Errorf("creating eventlog-Get-WinEvent.json: %w", err)
+	}
+
+	filterExpression := fmt.Sprintf(`'@{LogName='System'; ProviderName='Service Control Manager'; Data=%s}'`, kolideSvcName)
```

Comment:
```suggestion
	filterExpression := fmt.Sprintf(`@{LogName='System'; ProviderName='Service Control Manager'; Data='%s'}`, kolideSvcName)
```

Sorry, this is what I meant!

Reply:
:smacks forehead: of course.